packages/plugins/src/cli/deploy.ts

```diff
@@ -304,6 +305,11 @@ export const deploy = async (
     initCodeWithArgsArray
   )
 
+  assertContractSizeLimitNotExceeded(
+    deploymentInfo.accountAccesses,
+    configArtifacts
+  )
+
```

Comment:
Nit: This is somewhat outside the scope of this issue, but it would be nice if we just had a single function that does all checks like this (i.e., it includes the `assertNoLinkedLibraries` call below). That seems more maintainable to me.

Reply:
We're going to remove `assertNoLinkedLibraries` when we support linked libraries, so I don't think it's worth refactoring this just yet